src/main/java/openfasttrack/importer/markdown/MarkdownImporterFactory.java

```diff
@@ -22,17 +22,17 @@
  * #L%
  */
 
-
 import java.io.Reader;
 
 import openfasttrack.importer.ImportEventListener;
 import openfasttrack.importer.Importer;
 import openfasttrack.importer.ImporterFactory;
+import openfasttrack.importer.RegexMatchingImporterFactory;
 
 /**
  * {@link ImporterFactory} for Markdown files
  */
-public class MarkdownImporterFactory extends ImporterFactory
+public class MarkdownImporterFactory extends RegexMatchingImporterFactory
```

Comment:
Is this name still correct given that the configuration accepts globs too?

Reply:
PatternMatchingImporterFactory maybe?

Reply:
No. `RegexMatchingImporterFactory` only accepts regexp `Pattern`s. Only `LegacyTagImporterFactory` also supports globs.

Reply:
Ok.